src/ACadSharp/IO/DWG/DwgStreamWriters/DwgHandleWriter.cs

```diff
@@ -30,7 +30,7 @@ public DwgHandleWriter(ACadVersion version, MemoryStream stream, Dictionary<ulon
 				this._handleMap.Add(item.Key, item.Value);
 			}
 #else
-			this._handleMap = new Dictionary<ulong, long>(map.OrderBy(o => o.Key));
```

Comment:
I think this fixed some issues for the dwg output, it cannot be removed.

Reply:
Can I get any example dwg file that faces issues with this being not changed?
I am doubting that because, you are creating a new Dictionary. The order of the collection that you are providing shouldn't have any effect because dictionaries are unordered collections.
If you can give me a dwg file with the issues mentioned, then I can dig deep into it.